thrills-Test.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>paul's playground</title>
+
+	<script defer type=importmap-shim src="/x/importmap.json"></script>
+	<script defer type=module-shim src="/x/mock.js"></script>
+	<script defer src="/node_modules/es-module-shims/dist/es-module-shims.js"></script>
+
+	<style>
+		.container {
+			max-width: 1440px;
+			margin: auto;
+			padding: 10vh 0;
+		}
+
+		namaka-review-comment {
+			margin: auto;
+		}
+	</style>
+</head>
+<body>
+	<div class="container">
```

Comment:
let's go ahead and place this onto the regular `index.html`

and let's make sure the component is visually adaptive, so it works on both dark theme and light theme.